scripts/generate_schemas.py

```diff
@@ -131,6 +132,15 @@ def formatRequiredFields(self):
         return string
 
     def writeConstructor(self, outputFile):
+        # Force using slots to avoid the overhead of a dict per object;
+        # when a query returns hundreds of thousands of calls this can
+        # save a hundred megabytes or more.
+        print("    __slots__ = ['",
+              textwrap.fill(
```

Comment:
Nice! Wish I'd known about textwrap earlier. I've filed an issue saying we should update the rest of this script to follow the same pattern.